micrometer-core/src/main/java/io/micrometer/core/instrument/config/MeterRegistryConfigValidator.java

```diff
@@ -54,7 +56,7 @@ public static <M extends MeterRegistryConfig> Validated<?> checkAll(M config,
      * @return A function which, given a configuration instance, validates the property.
      */
     @SuppressWarnings("unchecked")
-    public static <M extends MeterRegistryConfig, T> Function<M, Validated<T>> check(String property,
+    @NullUnmarked public static <M extends @NonNull MeterRegistryConfig, T> Function<M, Validated<T>> check(@NonNull String property,
```

Comment:
I found this as a workaround for the error we had.
I also tied to help the compiler with explicit types (refactoring lambdas to `Function<...>` variables) without much luck.

Also opened https://github.com/micrometer-metrics/micrometer/issues/6404

Reply:
Also see: https://github.com/spring-io/spring-javaformat/issues/446